src/lib-components/indicators/InlineAlert.vue

```diff
@@ -0,0 +1,200 @@
+<script setup lang="ts">
+import { computed, ref } from "vue"
+
+import {
+  CheckCircleIcon,
+  InformationCircleIcon,
+  ExclamationIcon,
+  XCircleIcon,
+} from "@heroicons/vue/solid"
+
+import { XIcon } from "@heroicons/vue/outline"
+
+const props = withDefaults(
+  defineProps<{
```

Comment:
There's a bit of deviation from existing patterns here on the props.  

- I preferred `type` over `kind`, but am ultimately indifferent if there are strong opinions in either direction.
- Shortening `primaryBtnLink` to `btnLink`, etc. was preferred as I'm mostly anticipating a single CTA on alerts when used.